app/src/app/store/mapStore.ts

```diff
@@ -69,6 +78,7 @@ export const useMapStore = create<MapStore>((set) => ({
     set((state) => {
       if (!state.mapRef) return { selectedLayer: null };
       addBlockLayers(state.mapRef, layer);
+      state.addVisibleLayerIds([BLOCK_LAYER_ID, BLOCK_HOVER_LAYER_ID]);
```

Comment:
I also don't love adding in these hard coded layer ids like this- shouldn't it follow the `layer` argument that's passed in somehow?

Reply:
I think it's okay for now since this is for the special block ids. Refactored into a subscribe